README.md

```diff
@@ -3,6 +3,25 @@
 Contains the source code of the [SANsheets](https://chrome.google.com/webstore/detail/santiment-data/khglcgdkikfpccnfonmimpfkmolokbbk?utm_source=permalink)
 google spreadsheet add-on.
 
+## Table of contents
+
+- [Table of contents](#table-of-contents)
+  - [Usage](#usage)
+  - [Development](#development)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+  - [Structure](#structure)
+    - [Queries](#queries)
+    - [Functions](#functions)
+    - [Sheet Functions](#sheet-functions)
+    - [Generators](#generators)
+  - [Running tests](#running-tests)
+  - [Running integration tests](#running-integration-tests)
+  - [Standart template tests](#standart-template-tests)
+  - [Docker tests](#docker-tests)
```

Comment:
These are not docker tests. Move to subsection of Running Tests. Call it - "Run tests in docker container"